Source/Core/Core/State.cpp

```diff
@@ -74,7 +74,7 @@ static Common::Event g_compressAndDumpStateSyncEvent;
 static std::thread g_save_thread;
 
 // Don't forget to increase this after doing changes on the savestate system
-constexpr u32 STATE_VERSION = 124;  // Last changed in PR 9097
+constexpr u32 STATE_VERSION = 125;  // Last changed in PR 9348
```

Comment:
This looks like a result of removing the debug interface. Are there any other functional changes in this refactor?